sim/team-validator.ts

```diff
@@ -1186,7 +1186,7 @@ export class TeamValidator {
 		}
 
 		if (
-			tierTemplate.isNonstandard === 'Pokestar' && dex.gen === 5 ||
```

Comment:
This allows using Pokestar mons in Gen6.

Reply:
done i think